docs/src/main/sphinx/connector/prometheus.md

```diff
@@ -33,6 +33,7 @@ prometheus.uri=http://localhost:9090
 prometheus.query.chunk.size.duration=1d
 prometheus.max.query.range.duration=21d
 prometheus.cache.ttl=30s
+prometheus.auth.http.header.name=Authorization
```

Comment:
Let's remove this line from the example. Listing in "Configuration properties" seems sufficient.